HLTriggerOffline/Btag/interface/HLTBTagHarvestingAnalyzer.h

```diff
@@ -48,6 +49,8 @@ class HLTBTagHarvestingAnalyzer : public DQMEDHarvester {
 			// Histogram handler
 			std::map<std::string, MonitorElement *> H1_;
 
+			std::vector<std::string> modules_;
```

Comment:
I need a string to construct the path, for example:
https://github.com/natalia-korneeva/cmssw/blob/HEP17-btaghlt-validation/HLTriggerOffline/Btag/src/HLTBTagHarvestingAnalyzer.cc#L55